Duplicati/Server/webroot/ngax/index.html

```diff
@@ -26,8 +26,9 @@
     <meta name="theme-color" content="#ffffff">
 
     <link rel="stylesheet" type="text/css" href="styles/smoothness/jquery-ui.min.css">
-    <link rel="stylesheet" type="text/css" href="styles/style.css?v=2.0.0.7">
-    <link rel="stylesheet" type="text/css" href="styles/themes.css?v=2.0.0.7">
+    <link rel="stylesheet" type="text/css" href="styles/{{active_theme}}.css">
+	<link rel="stylesheet" type="text/css" href="styles/default.css" disabled="disabled">
+	<link rel="stylesheet" type="text/css" href="styles/dark.css" disabled="disabled">	
```

Comment:
Can we use spaces for the indentation to be consistent with the surrounding code?

Reply:
Done.